packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/rules-1.14/alertmanager.rules.yaml.patch

```diff
@@ -8,8 +8,8 @@
  apiVersion: monitoring.coreos.com/v1
  kind: PrometheusRule
  metadata:
-@@ -212,4 +213,5 @@
-         {{ toYaml .Values.defaultRules.additionalRuleLabels | nindent 8 }}
```

Comment:
Do we need to keep these default rule relabellings?

Reply:
We do but they added better support for these in upstream so we don't need to update those ourselves anymore: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/rules-1.14/alertmanager.rules.yaml#L51-L58